uniflow/flow/config.py

```diff
@@ -186,6 +186,93 @@ def check_labels_in_label2score(self) -> bool:
+                generated_anser="Modern and more complex forms of operating systems began to emerge in the early 1960s.",
+                explanation="Both answers are equally good as they accurately pinpoint the early 1960s as the period when modern operating systems began to develop.",
+                label="Equivalent",
+            ),
+            Context(
+                context="Operating systems did not exist in their modern and more complex forms until the early 1960s. Hardware features were added, that enabled use of runtime libraries, interrupts, and parallel processing.",
+                question="What features were added to hardware in the 1960s?",
+                grounding_answer="Hardware in the 1960s saw the addition of features like runtime libraries and parallel processing.",
+                generated_anser="The 1960s saw the addition of fault tolerance and compatibility in hardware.",
+                explanation="The generated answer is worse because it inaccurately suggests the addition of capabilities of hardware in 1960s which is not supported by the context.",
+                label="Reject",
+            ),
+            Context(
+                context="Operating systems did not exist in their modern and more complex forms until the early 1960s. When personal computers became popular in the 1980s, operating systems were made for them similar in concept to those used on larger computers.",
+                question="When did operating systems in personal computer were similar to those used on larger computers?",
+                grounding_answer="In 1980s.",
+                generated_anser="In the early 1960s.",
+                explanation="The generated answer is much worse as it incorrectly states the early 1960s as the period of popularity for personal computers, contradicting the context which indicates the 1980s.",
+                label="Strong reject",
+            ),
+        ],
+    )
+
+    def __post_init__(self):
+        """Post-initialization to perform label check."""
+        for example in self.guided_prompt_template.examples:
+            if example.label.lower() not in [k.lower() for k in self.label2score]:
+                raise ValueError(
+                    "Inconsistent labels found in guided_prompt_template examples, "
+                    f"example label {example.label} not in label2score has keys {list(self.label2score.keys())}",
+                )
+
+    def check_labels_in_label2score(self) -> bool:
+        """
+        Check if every label in the guided_prompt_template's examples is a key in label2score.
+
+        Returns:
+            bool: True if all labels are in label2score, False otherwise.
+        """
+        for example in self.guided_prompt_template.examples:
+            if example.label not in self.label2score:
+                return False
+        return True
```

Comment:
nit: you can refactor `__post_init__` and `check_labels_in_label2score` to the base RaterConfig for proper OOP design without duplicate code across multiple child class.